libs/features/unused-guid-search/src/unused-guid-search.component.html

```diff
@@ -0,0 +1,52 @@
+<div class="container-fluid">
+  <div class="content-block">
+    <form [formGroup]="form" (ngSubmit)="onSearch()" class="mb-3">
+      <div class="form-group row">
+        <label class="col-sm-2 col-form-label">Table</label>
```

Comment:
We could put all the strings in en.json and leave them open for potential translations